src/main/java/frc/robot/RobotContainer.java

```diff
@@ -192,9 +193,19 @@ public void putShuffleboardCommands() {
         }
         if (Constants.FEEDER_DEBUG) {
             SmartShuffleboard.putCommand("Feeder", "Feed", CommandUtil.logged(new StartFeeder(feeder)));
+            SmartShuffleboard.putCommand("Feeder", "IntakeFeederCombo", CommandUtil.sequence(
+                    "IntakeFeederCurrentCombo",
+                    new SpoolIntake(intake,0.25),
```

Comment:
this 0.25 now appears in so many places. Constant?

Reply:
done